src/index.json

```diff
@@ -37,11 +37,11 @@
         ],
         "azure-cli-iot-ext": [
             {
-                "filename": "azure_cli_iot_ext-0.1.1-py2.py3-none-any.whl",
-                "sha256Digest": "28f5565fa0367da4694223bb8dfacd68be5213e6a49b3a463a10c2379c39da19",
-                "downloadUrl": "https://files.pythonhosted.org/packages/8e/69/6a10aa6be2ad1a054d874d93bd43c09aaca26a137bd0f8f961f03a249a9f/azure_cli_iot_ext-0.1.1-py2.py3-none-any.whl",
+                "filename": "azure_cli_iot_ext-0.3.1-py2.py3-none-any.whl",
+                "sha256Digest": "6932b46e39b5506b460ddc58723d9b2b2223e2a8894d2adda3b0007b4cdef445",
+                "downloadUrl": "https://github.com/Azure/azure-iot-cli-extension/releases/download/v0.3.1/azure_cli_iot_ext-0.3.1-py2.py3-none-any.whl",
```

Comment:
@digimaun Do you want to append to the list rather than overwrite the prev. value?
This is a list of extensions then the CLI has logic to filter the list and only install what's supported.
As this stands, once merged, a user on an older version of the CLI would not be able to install the old version of this extension.
If that's intended, it's fine to leave this as is.

Reply:
It's intended. The original v0.1.1 package should not be used at all anymore.

Reply:
Okay thanks.